config/aws_citc.py

```diff
@@ -0,0 +1,153 @@
+        {
+            'name': 'citc',
+            'descr': 'Cluster in the Cloud build and test environment on AWS',
+            'modules_system': 'lmod',
+    	    'hostnames': ['mgmt', 'login', 'fair-mastodon*'],
+            'prefix': f'reframe_logs/',
+            'partitions': [
+                {
+                    'name': 'c4.2xlarge-haswell',
+                    'access': ['--constraint shape=c4.2xlarge', '--export=NONE'],
+                    'descr': 'Haswell, 8 cores, 15 GiB',
+                },
+                {
+                    'name': 'c4.4xlarge-haswell',
+                    'access': ['--constraint shape=c4.4xlarge', '--export=NONE'],
+                    'descr': 'Haswell, 16 cores, 30 GiB',
+                },
+                {
+                    'name': 'c5a.2xlarge-zen2',
+                    'access': ['--constraint shape=c5a.2xlarge', '--export=NONE'],
+                    'descr': 'Zen2, 8 cores, 16 GiB',
+                },
+                {
+                    'name': 'c5a.4xlarge-zen2',
+                    'access': ['--constraint shape=c5a.4xlarge', '--export=NONE'],
+                    'descr': 'Zen2, 16 cores, 32 GiB',
+                },
+                {
+                    'name': 'c6a.2xlarge-zen3',
+                    'access': ['--constraint shape=c6a.2xlarge', '--export=NONE'],
+                    'descr': 'Zen3, 8 cores, 16 GiB',
+                },
+                {
+                    'name': 'c6a.4xlarge-zen3',
+                    'access': ['--constraint shape=c6a.4xlarge', '--export=NONE'],
+                    'descr': 'Zen3, 16 cores, 32 GiB',
+                },
+                {
+                    'name': 'c5.2xlarge-skylake-cascadelake',
+                    'access': ['--constraint shape=c5.2xlarge', '--export=NONE'],
+                    'descr': 'Skylake/Cascade lake, 8 cores, 16 GiB',
+                },
+                {
+                    'name': 'c5.4xlarge-skylake-cascadelake',
+                    'access': ['--constraint shape=c5.4xlarge', '--export=NONE'],
+                    'descr': 'Skylake/Cascade lake, 16 cores, 32 GiB',
+                },
+                {
+                    'name': 'c5d.2xlarge-skylake-cascadelake',
+                    'access': ['--constraint shape=c5d.2xlarge', '--export=NONE'],
+                    'descr': 'Skylake/Cascade lake, 8 cores, 16 GiB, 200GB NVMe',
+                },
+                {
+                    'name': 'c6i.2xlarge-icelake',
+                    'access': ['--constraint shape=c6i.2xlarge', '--export=NONE'],
+                    'descr': 'Icelake, 8 cores, 16 GiB',
+                },
+                {
+                    'name': 'c6g.2xlarge-graviton2',
+                    'access': ['--constraint shape=c6g.2xlarge', '--export=NONE'],
+                    'descr': 'Graviton2, 8 cores, 16 GiB',
+                },
+                {
+                    'name': 'c6g.4xlarge-graviton2',
```

Comment:
Maybe better to use something more descriptive, like `aarch64-graviton2-16c-32gb`?

Reply:
Done